browser-test/src/applicant/navigation/application_navigation_flow.test.ts

```diff
@@ -91,55 +91,6 @@ test.describe('Applicant navigation flow', () => {
       await logout(page)
     })
 
-    test.describe(
-      'review page with North Star enabled',
-      {tag: ['@northstar']},
-      () => {
-        test.beforeEach(async ({page}) => {
-          await enableFeatureFlag(page, 'north_star_applicant_ui')
-        })
-
-        test('validate screenshot', async ({page, applicantQuestions}) => {
-          await applicantQuestions.clickApplyProgramButton(programName)
-
-          await validateScreenshot(
```

Comment:
Note for reviewers: the North Star preview page will be deleted at some point, so this test is no longer needed.

Reply:
Update: We are **not deleting** the preview page in North Star. We will revise it (See https://github.com/civiform/civiform/issues/7935). Instead, I'm changing the tests to reflect the new behavior